staked-relayer/src/relay/mod.rs

```diff
@@ -18,6 +18,9 @@ pub async fn run_relayer(runner: Runner<Error, BitcoinClient, PolkaBtcClient>) -
             Err(CoreError::Issuing(Error::PolkaBtcError(ref err))) if err.is_duplicate_block() => {
                 tracing::info!("Attempted to submit block that already exists")
             }
+            Err(CoreError::Issuing(Error::PolkaBtcError(ref err))) if err.is_rpc_disconnect_error() => {
+                return Err(ServiceError::ClientShutdown);
```

Comment:
Why exit relayer rather than retry to establish a connection?

Reply:
The websocket client has disconnected, we handle the signal to shutdown as a restart.